pystarport/pystarport/proto_python/confio/proofs_pb2_grpc.py

```diff
@@ -0,0 +1,3 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
do we want to commit these protoc generated files for tests instead of generating them on the fly?

Reply:
i'll add grpc cli python utilities afterwards.
we can add eventcall via grpc, and other useful features as python codes in pystarport.